packages/fast-components-styles-msft/src/slider-label/index.ts

```diff
@@ -1,136 +1,128 @@
+import { DesignSystem } from "../design-system";
+import { neutralForegroundRest, neutralOutlineRest } from "../utilities/color";
+import { applyCursorDefault } from "../utilities/cursor";
+import { heightNumber } from "../utilities/density";
+import { designUnit } from "../utilities/design-system";
 import { applyScaledTypeRamp } from "../utilities/typography";
 
-const styles: ComponentStyles<SliderLabelClassNameContract, DesignSystem> = (
-    config: DesignSystem
-): ComponentStyleSheet<SliderLabelClassNameContract, DesignSystem> => {
-    const designSystem: DesignSystem = withDesignSystemDefaults(config);
-    const direction: Direction = designSystem.direction;
+function minMaxLabelMargin(config: DesignSystem): string {
+    return toPx(((heightNumber()(config) / 2 + designUnit(config)) / 2) * -1);
+}
 
-    return {
-        sliderLabel: {
-            display: "grid",
-        },
+const styles: ComponentStyles<SliderLabelClassNameContract, DesignSystem> = {
+    sliderLabel: {
+        display: "grid",
+        ...applyCursorDefault(),
+    },
 
-        sliderLabel_positioningRegion: {
-            display: "grid",
-        },
+    sliderLabel_positioningRegion: {
+        display: "grid",
+    },
 
-        sliderLabel_label: {
-            ...applyScaledTypeRamp("t9"),
-            whiteSpace: "nowrap",
-            color: neutralForegroundActive,
-        },
+    sliderLabel_label: {
+        ...applyScaledTypeRamp("t9"),
+        whiteSpace: "nowrap",
+        color: neutralForegroundRest,
+    },
 
-        sliderLabel_tickMark: {
-            background: neutralOutlineRest,
-        },
+    sliderLabel_tickMark: {
+        background: neutralOutlineRest,
+    },
 
-        sliderLabel__positionMin: {},
+    sliderLabel__positionMin: {},
 
-        sliderLabel__positionMax: {},
+    sliderLabel__positionMax: {},
 
-        sliderLabel__horizontal: {
-            "&$sliderLabel": {
-                alignSelf: "start",
-                gridRow: "2",
-                marginTop: "-2px",
-            },
+    sliderLabel__horizontal: {
+        "&$sliderLabel": {
+            alignSelf: "start",
+            gridRow: "2",
+            marginTop: "-2px",
+        },
 
-            "& $sliderLabel_positioningRegion": {
-                gridTemplateColumns: "0",
-                gridTemplateRows: "auto auto",
-                width: "0",
-                justifySelf: "center",
+        "& $sliderLabel_positioningRegion": {
+            gridTemplateColumns: "0",
+            gridTemplateRows: "auto auto",
+            width: "0",
+            justifySelf: "center",
 
-                "&$sliderLabel__positionMin": {
-                    "& $sliderLabel_label": {
-                        marginLeft: "-10px",
-                        justifySelf: "start",
-                    },
+            "&$sliderLabel__positionMin": {
+                "& $sliderLabel_label": {
+                    marginLeft: minMaxLabelMargin,
```

Comment:
RTL? And below.

Reply:
There's a separate `sliderLabel__rtl` section below. That's how it was built originally, which might be a good practice for RTL support in general as it keeps the style sheet clean.